src/components/dropdown/react/Dropdown.tsx

```diff
@@ -62,6 +62,7 @@ const CLASSNAME: string = getRootClassName(COMPONENT_NAME);
 const DEFAULT_PROPS: IDefaultPropsType = {
     closeOnClick: true,
     escapeClose: true,
```

Comment:
Rename to `closeOnEscape` to uniformize with `closeOnClick` ?

Reply:
Good point !